tree-sitter-python/test/Manual/Examples.hs

```diff
@@ -1,18 +1,27 @@
 {-# LANGUAGE DisambiguateRecordFields, OverloadedStrings, OverloadedLists, TemplateHaskell #-}
-module Main (main) where
+module Manual.Examples (tests) where
 
 import           Control.Monad.IO.Class
-import           Data.Bool (bool)
 import           Data.ByteString (ByteString)
-import           Data.List.NonEmpty (NonEmpty(..))
+import           Data.List.NonEmpty
 import           GHC.Generics
 import           Hedgehog
-import           System.Exit (exitFailure, exitSuccess)
+import           Hedgehog.Internal.Property (PropertyName (..))
+import           Test.Tasty
+import           Test.Tasty.Hedgehog as Hedgehog
 import           TreeSitter.Python
 import qualified TreeSitter.Python.AST as Py
 import           TreeSitter.Token
 import           TreeSitter.Unmarshal
 
+tests :: TestTree
+tests = testGroup "(checked by Hedgehog)" props
+  where
+    props = let (Group _ xs) = $$(discover)
+            in fmap toP xs
+    toP ((PropertyName name), prop) = Hedgehog.testProperty name prop
```

Comment:
I did this to keep around the existing property tests. These might be redundant now. What do you think @aymannadeem and @maxbrunsfeld - should we remove them entirely?

Reply:
I'm not sure what properties they test. Is there overlap with what we're verifying here?

Reply:
I can't quite tell from this snippet what the delta between the other tests vs. these properties are, but I've found property tests super helpful in shedding light on blind spots historically 🤷‍♀

Reply:
If you expand the context 👇 below you can see. They aren't actually property tests, more like unit tests just using the hedgehog `Property`. It's stuff like:

``` haskell
"hello" `shouldParseInto` Py.Module { Py.ann = (), Py.extraChildren = [R1 identifier] }
```

... which is why I think it all might be covered by the corpus tests, but let's leave it for now and clean up in another pass.